dataland-frontend/src/components/general/ModalsComponent.vue

```diff
@@ -0,0 +1,102 @@
+<template>
+  <template v-if="_isModal()">
+    <component
+      :is="_findModalComponent().triggerComponent"
+      :component="_findModalComponent().dataComponent"
```

Comment:
*`displayComponent`?

Reply:
for both the prop `component` and the property of `FieldTriggerAndDataComponents` (which can be renamed on this way too).

Reply:
name changed to "displayComponent"

Reply:
Let's rename dataComponent to displayComponent as well.